elisp/log4slime.el

```diff
@@ -14,13 +14,64 @@
 ;;; See the License for the specific language governing permissions and
 ;;; limitations under the License.
 
-(require 'slime)
 (require 'cl-lib)
 (require 'log4cl)
+(require 'slime)
 
 (declare-function log4slime-highlight-log-message "log4cl.el")
 (declare-function slime-repl-emit "ext:slime-repl.el")
 
+(defvar log4slime-backend)              ; From log4cl.el
```

Comment:
Why are the ones that come from `log4cl` needed? I thought `require` would enforce the compile/load order.